subtitles.go

```diff
@@ -327,7 +339,9 @@ func (l Line) String() string {
 	for _, i := range l.Items {
 		texts = append(texts, i.Text)
 	}
-	return strings.Join(texts, " ")
+	t := strings.Join(texts, " ")
+	t = strings.ReplaceAll(t, "  ", " ")
```

Comment:
If you're trying to replace multi spaces into single spaces, I think you'd be better off using a regexp

Reply:
done